Makefile.main

```diff
@@ -106,11 +106,17 @@ docker-push:
 		repository=`echo $${d} | cut -d":" -f 2`; \
 		echo "Building dockerfile $$dockerfile for repository $$repository."; \
 		docker build -t $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$${repository}:$(TAG) -f $(WORKDIR)/$$dockerfile . && \
+		docker tag $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$${repository}:$(TAG) $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$${repository}:latest && \
 		docker push $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$${repository}:$(TAG); \
 		if [ $$? != 0 ]; then \
 			echo "Something went wrong pushing the image."; \
 			exit 1; \
 		fi; \
+		docker push $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$${repository}:latest; \
```

Comment:
I suggest do this in another rule, and this rule only be used when you are in Travis.

Reply:
Great idea! Thanks!
I'll do so this evening!